msteams-platform/sbs-file-handling-in-bot.yml

```diff
@@ -0,0 +1,306 @@
+    
+    **To add any necessary API permissions for downstream calls**
+    1. Select **API permissions** from the left pane.
+    1. Select **Add a permission**. **Request API permissions** window appears.
+    1. Select **Microsoft API's**. Then select **Microsoft Graph** and select **Delegated permissions**.
+    1. Add any user **Delegated permissions** that your app requires to downstream APIs, for example, User.Read.</br>
+
+        ![API permissions](~/assets/images/adaptive-cards/add-permissions.png)
+
+    **To enable implicit grant**
+    
+    1. Select **Authentication** from the left pane.
+    1. Select **Access tokens** and **ID tokens** checkboxes.</br>
+        ![Authentication Tokens](~/assets/images/adaptive-cards/authentication-tokens.png)
+    1. Select **Save** to save the changes.
+
+    **To update manifest**
+    1. Select **Manifest** from the left pane.
+    1. Ensure the config item is set to **"accessTokenAcceptedVersion": 2**. If not, change its value to **2** and select **Save**.
+        
+        > [!NOTE]
+        > If you are already testing your bot in Teams, you must sign out this app and sign out Teams. Then sign in again to see this change.
+
+        ![Update Manifest](~/assets/images/adaptive-cards/manifest.png)
+
+    **To set up Bot Service connection**
+
+    1. Go to **Home** page and select **Configuration** under **Settings** from the left pane.
+    1. Select **Add OAuth Connection Settings**.
+    1. In **New Connection Setting**, enter the following details:
+
+        | Field | Value or description |
+        | ----- | ----- |
+        | Name | Enter name for your new connection setting. You can use the name in the settings of your bot service code. |
+        | Service Provider | Select **Azure Active Directory V2**. |
+        | Client id | Saved previously as your **Microsoft App ID**. |
+        | Client secret | Saved previously as **Value** of the client secret ID. |
+        | Token Exchange URL | Use the **Application ID URL** obtained previously while exposing API endpoint. |
+        | Tenant ID | Enter **common**. |
+        | Scopes | Enter **User.Read** and also add all the **Scopes** required while specifying permissions to downstream APIs. |
+
+        :::image type="content" source="./assets/images/adaptive-cards/oauth-connection-setting.png" alt-text="Bot service connection":::
+
+    1. Select **Save**.
+
+- title: Test your app
+  durationInMinutes: 2
+  content: |
+
+    The most comprehensive way to test your app is by creating an app package and uploading it to Teams. This method is the only way to test the full functionality available to your app,
+    across all scopes. You can configure and run the Teams app by uploading the manifest.zip file.
+
+    To run the Teams file bot you need to upload the manifest.zip file in Teams.
+
+    1. In the File Explorer, go to botbuilder-samples > samples > csharp_dotnetcore > 56.teams-file-upload > appsettings.
```

Comment:
Bold File Explorer. Make this change globally.

Reply:
Made the changes.